examples/dt/neutron/CRD_files/neutron_az_ovn.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: core.openstack.org/v1beta1
+kind: OpenStackControlPlane
+metadata:
+  name: openstack
```

Comment:
I think the content needs to be structured to kustomize uses, but that's someone from maintainers can confirm.

Reply:
Unified team will take care of the Kustomize format